### PR TITLE
Gulp tasks

### PR DESCRIPTION
**Pull request by aintgoin2goa**
These tasks, when applied to apps, should allow us to have sourcemaps in production

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Financial-Times/next-gulp-tasks.git"
+  },
+  "author": "",
+  "license": "MIT",
```
Comment (matthew-andrews) [inline package.json]: nope :

**Comment by matthew-andrews** [comment]
nice! :+1:

File: package.json (hunk 1 above)
Comment (aintgoin2goa) [inline package.json]: ?

Comment (matthew-andrews) [inline package.json]: MIT

Comment (aintgoin2goa) [inline package.json]: what should it be?  Was basing it on this:
https://docs.google.com/a/ft.com/document/d/1pI-qI3BrO5edFYdHcoDCH9wVbfbFvroclxSEtkXwpCw/edit#heading=h.ofupfotthc95

**Pull request merged**